activesupport/activesupport.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |s|
 
   s.rdoc_options.concat ['--encoding',  'UTF-8']
 
-  s.add_dependency('i18n',       '~> 0.6', '>= 0.6.4')
+  s.add_dependency('i18n',       '~> 0.6.0', '>= 0.6.4')
```

Comment:
@simi no it is not, you should cap the upper bound too, if not it will go all the way above 0.6